src/components/Subthought.tsx

```diff
@@ -114,6 +116,34 @@ const Subthought = ({
     ref.current.style.opacity = opacity
   })
 
+  /**
+   * Cleans up editable classes from the provided HTML string.
+   *
+   * @param htmlString - The HTML string to clean up.
+   * @returns The cleaned HTML string.
+   */
+  const cleanUpEditableClasses = (htmlString: string) => {
+    const container = document.createElement('div')
+    container.innerHTML = htmlString
+    const editableElements = container.querySelectorAll('.editable--preventAutoscroll_true')
+    editableElements.forEach(element => {
+      element.classList.remove('editable--preventAutoscroll_true')
+    })
+    return container.innerHTML
+  }
+
+  // Capture the static HTML string when the thought is first rendered
+  useEffect(() => {
+    if (thought && ref.current) {
+      cachedHTMLRef.current = cleanUpEditableClasses(ref.current.innerHTML)
+    }
+  }, [thought, ref])
```

Comment:
What do you think about putting the `useEffect`, `cleanUpEditableClasses`, and refs in a custom hook? That might help keep this component tidy.